app/web/src/store/auth.store.ts

```diff
@@ -150,6 +150,9 @@ export const useAuthStore = defineStore("auth", {
         if (errCode === "WORKSPACE_NOT_INITIALIZED") {
           // db is migrated, but workspace does not exist, probably because it has been reset
           const _reconnectReq = await this.AUTH_RECONNECT();
+          // WHAT HAD HAPPENED WAS...
+          // in a local scenario where prd auth portal has a workspace, and local does not, the first attempt this will hit an infinite loop trying to find the workspace, totally stuck
+          // on a second attempt, it appears, the workspace gets created and user exits the loop
           // TODO: react to failure here?
```

Comment:
This is messy. Good comment, but it feels like there's a longer term fix waiting here once we pay more attention to the auth stack again.

Reply:
Goodness I know, we found the TODO when deving this out + decided to add a bit more colour as it really wasn't clear what was going on.

I am genuinely not sure what the long term fix would be here apart from the obvious response of "handle this case"

Reply:
That's exactly the fix. We'll do it.